### DISCUSSION TITLE
Does flask have any EOL policy?

### DISCUSSION THREAD

**Discussion by haleng82** [pallets/flask#4688] [Q&A]
Does flask have any End of Life policy?

**Comment by davidism** [comment]
The latest feature branch gets bug fixes. That's currently 2.1.x. When a new feature release is made, that will become the new supported branch. Always use the latest release to get the latest features and fixes. Applications should pin their dependency tree (not just Flask) using one of the many available tools to control when they get updates. This is a common strategy for open source projects.
